src/main/java/io/reactivex/Maybe.java

```diff
@@ -564,6 +564,42 @@
     }
 
     /**
+     * Wraps a Completable into a Maybe.
+     *
+     * <dl>
+     *  <dt><b>Scheduler:</b></dt>
+     *  <dd>{@code fromCompletable} does not operate by default on a particular {@link Scheduler}.</dd>
+     * </dl>
+     * @param <T> the target type
+     * @param completable the completable to convert from
+     * @return the new Maybe instance
+     * @throws NullPointerException if completable is null
+     */
+    @SchedulerSupport(SchedulerSupport.NONE)
+    public static <T> Maybe<T> fromCompletable(final Completable completable) {
```

Comment:
Yeah I copied the signature from the method above and that one is using final. Not everything is consistent though. I'll change it though